utils/build-script-impl

```diff
@@ -2073,14 +2086,12 @@ for host in "${ALL_HOSTS[@]}"; do
                   echo "Cleaning the XCTest build directory"
                   call rm -rf "${XCTEST_BUILD_DIR}"
 
-                  LLVM_BIN="$(build_directory_bin ${LOCAL_HOST} llvm)"
-
                   cmake_options=(
                     ${cmake_options[@]}
                     -DCMAKE_BUILD_TYPE:STRING="${XCTEST_BUILD_TYPE}"
-                    -DCMAKE_C_COMPILER:PATH="${LLVM_BIN}/clang"
-                    -DCMAKE_CXX_COMPILER:PATH="${LLVM_BIN}/clang++"
-                    -DCMAKE_Swift_COMPILER:PATH="$(build_directory_bin ${LOCAL_HOST} swift)/swiftc"
+                    -DCMAKE_C_COMPILER:PATH="${CLANG_BIN}/clang"
+                    -DCMAKE_CXX_COMPILER:PATH="${CLANG_BIN}/clang++"
+                    -DCMAKE_Swift_COMPILER:PATH=${SWIFTC_BIN}
```

Comment:
This was set this way before but I notice that [@benlangmuir changed it](https://github.com/apple/swift/commit/5619138a10ef#diff-65b44eb6cb88af2161d8e1176231aad0L2658), a mistake? Would be good if you could review this pull.